frontend/src/js/tooltip/AdditionalInfoHoverable.tsx

```diff
@@ -13,32 +14,39 @@ const Root = styled("div")`
 `;
 
 // Allowlist the data we pass (especially: don't pass all children)
-const getAdditionalInfos = (node: ConceptT): AdditionalInfosType => ({
+const getAdditionalInfos = (
+  node: ConceptT,
+  parent?: string,
+): AdditionalInfosType => ({
   label: node.label,
   description: node.description,
   isFolder: !!node.children && node.children.length > 0,
   matchingEntries: node.matchingEntries,
   matchingEntities: node.matchingEntities,
   dateRange: node.dateRange,
   infos: node.additionalInfos,
+  parent: parent ? getConceptById(parent)?.label : null,
```

Comment:
What's the performance impact from that? Like this, every time someone hovers, we'll do a lookup and that's probably not instant.

I'd suggest to 
a) at least pass the root id a second param as well for optimized lookup, see implementation of the function
b) better even: do the optimized root lookup when the ConceptTreeNode renders and pass the root `label` directly, instead of the root id, maybe even re-use the looked up root in `createQueryElement`